kf_lib_data_ingest/common/misc.py

```diff
@@ -116,6 +117,16 @@ def numeric_to_str(val, replace_na=False, na=None):
 
     :return: new representation of `val`
     """
+    if isinstance(val, list):
+        # make hashable without changing style or losing comparability
+        return str(sorted(
```

Comment:
Why do you need to sort it? What if the order of the list mattered in the source data?

Reply:
I think the order of the list will never actually matter. In practice, all lists in data should probably be sets.

Reply:
Ok I see